packages/react-next/src/components/ContextualMenu/examples/ContextualMenu.Submenu.Example.tsx

```diff
@@ -66,15 +66,15 @@ const menuItems: IContextualMenuItem[] = [
     key: 'share',
     subMenuProps: {
       items: [
-        { key: 'sharetotwitter', text: 'Share to Twitter' },
-        { key: 'sharetofacebook', text: 'Share to Facebook' },
+        { key: 'shareToTwitter', text: 'Share to Twitter' },
```

Comment:
Can you revert this file as well, since the changes are non-essential and it would conflict with the example move PR?